README.md

```diff
@@ -123,6 +123,8 @@ It's a great way to learn.
 * [**Node.js**: _Create a CLI tool in Javascript_](https://citw.dev/tutorial/create-your-own-cli-tool)
 * [**Rust**: _Command line apps in Rust_](https://rust-cli.github.io/book/index.html)
 * [**Rust**: _Writing a Command Line Tool in Rust_](https://mattgathu.dev/2017/08/29/writing-cli-app-rust.html)
+* [**Python**: _Create a interactive commandline tool in python_](https://pratikpathak.com/python-poetry-example/#Lets_start_building_the_Project)
+
```

Comment:
I don't think this is link is placed under a fitting section to me. Correct me if I am wrong but this link seems more like how to use the Poetry cli tool than how to create it as required on this section, the section title clearly states "how to create your own cli tool"

Reply:
Hi @proxybee
This article is about building a command line tool using Poetry as a package manager. The title is a little bit misleading. I will make the changes and raise a new PR. Thankyou